example/App.js

```diff
@@ -0,0 +1,6 @@
+import "react-native-gesture-handler";
+import { registerRootComponent } from "expo";
+
+import App from "./src/index";
+
+registerRootComponent(App);
```

Comment:
There were like, four files before that essentially handled pieces of what this does now:

`__generated__/AppEntry`
`App.tsx`
`App.js`
`src/App.tsx`

Centralized here, simpler.